### ISSUE TITLE
Entry point widget for supply re-ordering

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#79823]
### User story

As a Veteran, I want the benefit-hub entry points for the authenticated supply-reordering form to require authentication before showing the link to the form, so that I can easily reach the form in its new authenticated location following the pattern I'm familiar with on VAgov.

### Notes
- [CAIA ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77499) for moving the entry point URLs – that CAIA ticket will be unblocked by this one
- This is a common pattern throughout the Benefit Hubs UX. See [CAIA docs](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md).
- [Example PR](https://github.com/department-of-veterans-affairs/vets-website/commit/b4f4e0eb439bf3a7f665df9f55f7ff1a21cd6924) from the 10-10 team
- [Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1711583333479549?thread_ts=1711405283.122249&cid=C52CL1PKQ) with Jill and Randi, wherein we determined it would be Cartographers' job to implement the widgets.

Possible tasks:
- [ ] Get content approved by CAIA (Sara might have done this already?)
- [x] implement widget
- [x] inform CAIA
- [ ] flip flag to "on" in prod
- [ ] CAIA publish page that leverages new widget
- [ ] verified in prod
- [ ] removed feature flag code

## Acceptance criteria

- Coordinated launch date with CAIA
- The benefit hub page for ordering hearing aid batteries, https://www.va.gov/health-care/order-hearing-aid-batteries-and-accessories/, asks unauthenticated users to sign in before revealing the link to the form
- The benefit hub page for ordering CPAP supplies, https://www.va.gov/health-care/order-cpap-supplies/, asks unauthenticated users to sign in before revealing the link to the form
- Feature flag tech debt has been removed from codebase

**Comment by radavis** [comment]
Blocked by: https://github.com/department-of-veterans-affairs/va.gov-team/issues/80007

**Comment by radavis** [comment]
Filed a support request at #cms-support: https://dsva.slack.com/archives/CDHBKAL9W/p1712669386373349

**Comment by radavis** [comment]
Met with Jill A. and Randi M. Going to attempt to feature toggle the ctaWidget on the vets-api/vets-website side.

**Comment by wesrowe** [comment]
pulled from S29 due to pivot in priorities

**Comment by radavis** [comment]
Blocked by: department-of-veterans-affairs/vets-website#29577

**Comment by radavis** [comment]
Blocked:
- Waiting on @department-of-veterans-affairs/va-platform-cop-frontend to review department-of-veterans-affairs/vets-website#29499

**Comment by wesrowe** [comment]
An observation from demo i thought I should mention – there's probably a good reason for it. I noticed that both the unauthed and authed versions of the widget appeared together in a screenshot. Probably an artifact of the toggling/handoff process...

**Comment by radavis** [comment]
Blocked. Ask has changed significantly since ticket creation.

**Comment by radavis** [comment]
ACs: https://dsva.slack.com/archives/C0581MN69TJ/p1716316075994109?thread_ts=1716229139.767859&cid=C0581MN69TJ

**Comment by wesrowe** [comment]
adding a link to Laura's latest [update](https://dsva.slack.com/archives/C0581MN69TJ/p1716411003083979?thread_ts=1716393514.001609&cid=C0581MN69TJ) [NOTE the sharepoint link]:
>here's the updated [widget content](https://dvagov-my.sharepoint.com/:w:/r/personal/laura_willwerth_va_gov/Documents/Sign-in%[%E2%80%A6]d=wce9c9573de694dbc93818d0709e4f1a8&csf=1&web=1&e=TR37yH) to prep for implementation
>
>Before the new widgets go live, we'll need to confirm that the new R&S page we link to from several widgets is published

**Comment by sterkenburgsara** [comment]
@radavis I'm behind on this / need updates - I'm logging onto the VA network now to pull the latest content updates. I'll circle back when I'm up to date on what is needed & have it in place for you.

**Comment by sterkenburgsara** [comment]
@laurwill can you give me access to this widget doc? looks like it's on your personal drive & I don't have access.

**Comment by wesrowe** [comment]
@radavis, FYSA that slack discussion was about a future version of widgets that are smarter. Content for the initial version of the sign-in alert is as you have implemented it. So I don't think you're blocked...

**Comment by LynneJohnson224** [comment]
Per slack thread, ticket is blocked due to:

1. We need to confirm the route guard decision before finalizing the entry point widget
2. We need SME approval of the Drupal page content before we can publish that page (Laura Willwerth to follow up with SME today)

**Comment by LynneJohnson224** [comment]
8/7/2024 Refinement Notes: Need to discuss this ticket with @wesrowe - should we continue with this ticket given new discovery/changes, etc or should we create a new ticket?

**Comment by wesrowe** [comment]
Let's remember also to consider the ticket in the current sprint, https://github.com/department-of-veterans-affairs/va.gov-team/issues/89275.

**Comment by wesrowe** [comment]
We are waiting to close this until Carlos's PR for #89275 is merged.

**Comment by wesrowe** [comment]
Closing. Per refinement 12/11, the ball is back in our court to update the widget. #[94862](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94862) takes over.
